packages/gvim.rb

```diff
@@ -86,11 +98,38 @@ def self.install
 
   def self.postinstall
     puts
-    puts "The config files are located in #{CREW_PREFIX}/etc".lightblue
-    puts 'User-specific configuration should go in ~/.gvimrc'.lightblue
+    puts "The config files are located in #{CREW_PREFIX}/etc.".lightblue
+    puts 'User-specific configuration should go in ~/.vimrc.'.lightblue
     puts
     puts 'If you are upgrading from an earlier version, edit ~/.bashrc'.orange
     puts "and remove the 'export VIMRUNTIME' and 'export LC_ALL=C' lines.".orange
     puts
+    # Set vim to be the default vi if there is no vi or if a default
+    # vi does not exist.
+    @crew_vi = File.file?("#{CREW_PREFIX}/bin/vi")
+    @system_vi = File.file?('/usr/bin/vi')
+    @create_vi_symlink = true if !@system_vi && !@crew_vi
+    @create_vi_symlink_ask = true if @crew_vi || @system_vi
+    if @create_vi_symlink_ask
+      print "\nWould you like to set vim to be the default vi [y/N] "
+      case $stdin.getc
+      when 'y', 'Y'
+        @create_vi_symlink = true
+      else
+        @create_vi_symlink = false
+        puts 'Default vi left unchanged.'.lightgreen
+      end
+    end
+    return unless @create_vi_symlink
+
+    FileUtils.ln_sf "#{CREW_PREFIX}/bin/vim", "#{CREW_PREFIX}/bin/vi"
+    puts 'Default vi set to vim.'.lightgreen
```

Comment:
Why are we symlinking gvim to vi?  Isn't vi a cli command?

Reply:
The gvim package also includes a cli vim binary. (gvim and vim are mutually exclusive.... gvim is basically vim also linked to gtk.)